spec/features/admin_split_decision_create_spec.rb

```diff
@@ -36,15 +36,13 @@
     end
 
     expect(split_page).to be_displayed
-    expect(split_page).to have_content "Queued decision"
-
-    Delayed::Worker.new.work_off
+    expect(split_page).to have_content "Decided"
 
     split_page.load split_id: split.id
     expect(split_page).to be_displayed
 
-    expect(Assignment.where(variant: "hammer_time").count).to eq 0
-    expect(Assignment.where(variant: "touch_this").count).to eq 5
+    expect(split.assignment_count_for_variant(:hammer_time)).to eq 0
+    expect(split.assignment_count_for_variant(:touch_this)).to eq 0
```

Comment:
Now assignments just hide themselves completely when a decision overrides them.

Reply:
Do we not show the count somewhere on the page? Might be nice to avoid reaching straight to the model code if something on `split_page` can tell us what we need.

Reply:
Yeah, that's a fair shout, but this is super Not My Test :) There's a lot of work I'd do to the admin screens and tests given the chance, but I feel like that might be a bridge too far for this PR.